core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
I didn't use media queries, but bind the window to resize and changed the variable accordingly.It works just fine though not as quick as media queries.

`angular.element($window).bind('resize', function() {
      var screenWidth = windowDimensionsService.getWidth();
      if (screenWidth <= 1280) {
        $scope.isSmallerWindow = true;
      } else {
         $scope.isSmallerWindow = false;
      }
      $scope.$apply();
    });`